### DISCUSSION TITLE
Exclude potentially resolved tickets from cell navigation

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5625] [Feature Requests]
Add an option to exclude the modified error cells from the error cell navigator.

Would be useful when there are a lot of ambiguity errors, but many of them are fixed though "Apply All", and you only want to see what's left to fix

Suggested by @grantfitzsimmons

**Comment by emenslin** [comment]
Not sure if this is the same issue but possibly related to #2009

**Comment by grantfitzsimmons** [comment]
These are unrelated. This request seeks an option to display only the fields that still contain errors after resolving a number of problems before saving.
